src/shared/scrapers/AUS/SA/index.js

```diff
@@ -37,7 +38,7 @@ const scraper = {
       const paragraph = $('.middle-column p:first-of-type').text();
       const { casesString } = paragraph.match(/been (?<casesString>\d+) confirmed cases/).groups;
       return {
-        state: scraper.state,
+        state: this.state,
```

Comment:
This is added automatically to everything returned by the scraper, no need to add it here